src/components/task/TaskPanel.tsx

```diff
@@ -0,0 +1,186 @@
+import { Card, CardActions, CardContent, Collapse, ExpansionPanelDetails, ExpansionPanelSummary, Grid, IconButton, Typography } from '@material-ui/core';
+import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { withStyles } from '@material-ui/styles';
+import { APITask, APITest, ConvertToAPITests } from 'evergreen.js/lib/models';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+export const StyledExpansionPanel = withStyles({
```

Comment:
I'm not 100% sure what I would change, but I feel like the sidebar could use some color or labels to help differentiate the different pieces of data. Things like task status, time taken, and display task hierarchy aren't really apparent. What do you think?

Reply:
I agree with that, at least for including the status. The time taken I'm not so sure about- none of the people I interviewed mentioned that as being a helpful piece of information, but maybe I'm just missing a use case. The reason I didn't add the display task hierarchy is because it is really not apparent to me how that works just looking at data returned from the API, and it's not displayed well enough in the current UI for me to understand it.